asreview/io/paper_record.py

```diff
@@ -169,16 +169,16 @@ def body(self):
 
     def todict(self):
         """Create dictionary from the record."""
-        label = self.label
-        if self.label is LABEL_NA:
-            label = None
+        included = self.included
+        if self.included == LABEL_NA:
+            included = None
         paper_dict = {
             "title": self.title,
             "abstract": self.abstract,
             "authors": self.authors,
             "keywords": self.keywords,
             "record_id": self.record_id,
-            "label": label,
+            "included": included,
```

Comment:
This looks problematic. Do we know where this method is used?

Reply:
Apparently nowhere, since it was completely broken..

Reply:
Confirmed. Please remove the method.

Reply:
Done